cyclops/query/omop.py

```diff
@@ -171,104 +174,74 @@ def _map_care_site_id(self, source_table: Subquery) -> Subquery:
 
         return source_table
 
+    @table_params_to_type(Subquery)
+    def provider(
+        self,
+        join: Optional[qo.JoinArgs] = None,
+        ops: Optional[qo.Sequential] = None,
+    ) -> Subquery:
+        """Query provider table using query operations.
+
+        Parameters
+        ----------
+        join: qo.JoinArgs, optional
+        ops: List[qo.Sequential], optional
+
+        Returns
+        -------
+        cyclops.query.interface.QueryInterface
+            Constructed query, wrapped in an interface object.
+
+        """
+        table = self.get_table(PROVIDER)
+        if join:
+            table = qo.Join(**join)(table)
```

Comment:
How come Join and other ops are being applied here and also in `QueryInterface`? I'm confused because it doesn't fit the same pattern as other methods.

Reply:
Oh yes, i left it in there by mistake.